src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+    >
```

Comment:
You need to check and fix it everywhere
<img width="1079" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/931f9970-7659-4d2b-ba85-b6aa0ee0532f">